### PR TITLE
Fix pre-condition in chain/enqueue

### PR DESCRIPTION
**Pull request by ordnungswidrig**
Small fix for the precondition declaration on chain/enqueue

### CODE REVIEW

**Comment by ohpauleez** [comment]
I'm not sure why the service-template tests are failing on this change, but all tests should be fixed before this gets merged.  Additionally, we should add tests that exercise the precondition.

**Comment by ddeaguiar** [comment]
@ohpauleez the template tests are failing due to an issue with lein version 2.8.1. Refer to #558 which has been merged to master.

**Comment by ordnungswidrig** [comment]
@ohpauleez I'll happily add some tests but the interceptor project is lacking tests alltogether. Is there a separate place for the tests?

**Comment by ohpauleez** [comment]
@ordnungswidrig All tests for the entire project live in `pedestal.service`.  Probably makes the most sense to put them in the [base interceptor test file](https://github.com/pedestal/pedestal/blob/master/service/test/io/pedestal/interceptor_test.clj)

Thanks all!

**Comment by ordnungswidrig** [comment]
@ohpauleez test added.

File: service/test/io/pedestal/interceptor_test.clj [hunk 1]
```diff
@@ -196,6 +196,12 @@
                                  (catcher :h)])
                        :leave))))
 
+(deftest t-enque-precondition
```
Comment (ddeaguiar) [inline service/test/io/pedestal/interceptor_test.clj]: @ordnungswidrig can you add a test with a valid interceptor and drop the call to `execute`? It's not needed to test the precondition.

**Comment by ddeaguiar** [comment]
@ordnungswidrig looks good. Merging, thanks :)

**Pull request merged**